conda/recipes/libcuspatial/meta.yaml

```
@@ -86,12 +106,24 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
```

Should we have the compilers here?

```suggestion
      build:
        - {{ compiler('c') }}
        - {{ compiler('cxx') }}
        {% if cuda_major == "11" %}
        - {{ compiler('cuda11') }} ={{ cuda_version }}
        {% else %}
        - {{ compiler('cuda') }}
        {% endif %}
        - sysroot_{{ target_platform }} {{ sysroot_version }}
```

---

No, we have already finished compilation by the time we're packaging libcuspatial and libcuspatial-tests. Compilers are needed in the top-level `build` section for use by `build.sh`, but the individual package outputs only install (copy) files to the conda environment and do not perform compilation.

The only argument I could see for adding compilers would be to influence the host/run environments via `run_exports` but I think that is necessary here (we haven't traditionally done this).